src/locale/zh-tw.js

```diff
@@ -21,8 +21,8 @@ const locale = {
     LTS: 'HH:mm:ss',
     L: 'YYYY/MM/DD',
     LL: 'YYYY年M月D日',
-    LLL: 'YYYY年M月D日 HH:mm',
-    LLLL: 'YYYY年M月D日dddd HH:mm',
+    LLL: 'YYYY年M月D日Ah點mm分',
```

Comment:
can you please revert these two lines to make the same with https://github.com/moment/moment/blob/develop/src/locale/zh-tw.js

Reply:
OK, revert it.